lib/PuppeteerSharp/FrameManager.cs

```diff
@@ -68,14 +67,13 @@ void _client_MessageReceived(object sender, PuppeteerSharp.MessageEventArgs e)
                     OnLifeCycleEvent(e);
                     break;
             }
-
         }
 
         private void OnLifeCycleEvent(MessageEventArgs e)
         {
             if (Frames.ContainsKey(e.MessageData.frameId.ToString()))
             {
-                var frame = Frames[e.MessageData.frameId.ToString()];
+                Frame frame = Frames[e.MessageData.frameId.ToString()];
```

Comment:
I’d leave `var` here.

Reply:
The IDE thinks that it's type is `dynamic` and doesn't know we use the method ``